### ISSUE TITLE
Issue with pressing alt, ctrl, shift at the same time

### ISSUE THREAD

**Issue by K0JX** [wyatt-herkamp/too-many-shortcuts#15]
There is a problem with pressing 2 or 3 buttons (alt/ctrl/shift) at the same time. 
When pressing, for example: ctrl + shift + w + space at the same time and then letting go of either ctrl/shift/alt, then every other input in use is being cancelled so you stop moving and need to press w + space and every other key that you used again.

Tested on: 0.0.5 version
Minecraft version: 1.21 (Fabric Loader 0.15.11)
No other mods other than: Fabric API (0.102.0+1.21) and Fabric Language Kotlin (1.12.2+kotlin.2.0.20)

**Comment by wyatt-herkamp** [comment]
Thank you for the bug report. 

I have read and tested it. I see what you are talking about. However, I don't know how to fix it. 

What is happening is

A key release happens and triggers the active key binding to stop. What I will need to do is check if any kind of bindings match the new set of keys.

**Comment by X3zk** [comment]
is there a fix yet?

**Comment by wyatt-herkamp** [comment]
> is there a fix yet?

No, I am extremely busy with work and school and have no time to do anything.

**Comment by wyatt-herkamp** [comment]
I think I might have a fix for it

**Comment by wyatt-herkamp** [comment]
https://github.com/wyatt-herkamp/too-many-shortcuts/actions/runs/13530843726

Scroll to the bottom then download the mod file. You will need to extract it. But please test this and see if fixes the issue. This is for 1.21.4
